### ISSUE TITLE
Did you have the problem that this model runs well in simulate but got black pic with iOS device?

### ISSUE THREAD

**Issue by daxiangpanda** [lfarah/CoreML-Stylize#1]
Did you have the problem that this model runs well in simulate but got black pic with iOS device?

**Comment by lfarah** [comment]
Yeah.... I've been trying to fix this for the past 3 weeks but no fixes yet ☹️

**Comment by daxiangpanda** [comment]
oh.....so what kind of problem do you think in this model causing such problem?

**Comment by daxiangpanda** [comment]
do you know how google train that model ?

**Comment by lfarah** [comment]
Sorry... All I know is in the Readme 😢
